tests/system.py

```diff
@@ -736,6 +736,65 @@ def test_load_table_from_dataframe_w_automatic_schema(self):
         )
         self.assertEqual(table.num_rows, 3)
 
+    @unittest.skipIf(
+        pandas is None or pandas.__version__ < "1.0.1",
+        "Only `pandas version >=1.0.1` supports",
+    )
+    @unittest.skipIf(pyarrow is None, "Requires `pyarrow`")
+    def test_load_table_from_dataframe_w_none_value(self):
+        """Test that a DataFrame with dtypes that map well to BigQuery types
+        can be uploaded without specifying a schema.
```

Comment:
This comment is hard to understand

Reply:
Agreed, this one is a bit misleading and does not entirely agree with the test method name. It can be rephrased to something like
```
Test whether a DataFrame containing None-type values can be uploaded without specifying a schema.
```